### ISSUE TITLE
About using myself val_det

### ISSUE THREAD

**Issue by xiashh** [GengDavid/pytorch-cpn#2]
If I want to use myself detection results, should I transform the annotations to some particular format? I have read your code, and it seems that it's not mentioned.

**Comment by GengDavid** [comment]
Yes, if you want to use your own detection results, you'll need to transform your results into a json file. It should contain only one list of data. Each list item represents one detection result, and the format of it is like this:

```
{
    "unit": {
        "GT_bbox": [x1, y1, x2, y2]
    },
    "imgInfo": {
    	"imgID": int,
    	"img_paths": str,
    },
    "score": float
}

```
For example, for test image '000000581781.jpg' in test set, the imgID is 581781 and img_paths is '000000581781.jpg'. And score is the detection confidence value.  
The format may be a little bit strange because I only use ground truth bbox to test at the very beginning.

**Comment by xiashh** [comment]
Thank you very much!
I'm a novice for deep learning. So may I ask you about implementation pipeline?
I don't know where to begin.

**Comment by GengDavid** [comment]
This repo is implemented follow the implementation of [tf-cpn](https://github.com/chenyilun95/tf-cpn).   
And there is actually no fixed pipeline. Generally, you can build your network up first, and then you can train it with your own processed data. You can refer to others' hyper-parameter first to train your own networks.   
Have fun with it.

**Comment by xiashh** [comment]
Thank you!
